src/main/resources/com/google/api/codegen/go/sample.snip

```diff
@@ -14,6 +14,8 @@
     //    If not already done, install the gcloud CLI from
     //    https://cloud.google.com/sdk/ and run
     //    `gcloud beta auth application-default login`
+    //    For more information, see:
```

Comment:
I think we're currently consistent about using colons in forms like 
```
... see:
<URL>
```
so if we decide to change it here, it might be preferable to change elsewhere. I personally prefer the colon, but leaving it out seems okay.